src/main/java/org/ggp/base/util/game/LocalGameRepository.java

```diff
@@ -226,7 +226,10 @@ private static int getMaxVersionForDirectory(File theDir) {
                 } else if (theFile.getName().endsWith(".png")) {
                     // TODO: Handle other binary formats?
                     return readBinaryFile(theFile);
-                } else if (theFile.getName().endsWith(".xsl")) {
+                } else if (theFile.getName().endsWith(".jpg")) {
+                    return readBinaryFile(theFile);
+                } else if (theFile.getName().endsWith(".xsl"))
+                {

```

Comment:
Yeah, it's confusing to have one case in an if-else structure indented one way, and all of the other cases indented a different way. For the purposes of this change I suggest just leaving the opening { on the same line as the if clause, just like all of the other cases here.